build.js

```diff
@@ -0,0 +1,103 @@
+#! /usr/bin/env node
+const chalk = require('chalk')
+const tee = require('tee')
+const stripAnsi = require('strip-ansi')
+const { PassThrough } = require('stream')
+const readline = require('readline')
+const concurrently = require('concurrently')
+
+// Parse build flags and environment variables
+let watchMode = false
+let pointToLocalServer = false
+process.argv.slice(2).forEach(flag => {
+  switch (flag) {
+    case '--watch': return watchMode = true
+    case '--local-server': return pointToLocalServer = true
+    default: throw new Error(`Unrecognized flag "${flag}"`)
+  }
+})
+
+const watchFlag = watchMode ? '--watch' : ''
+const ENV = process.env.ENV || 'local'
+const ROAR_SERVER_URL = pointToLocalServer ? 'https://localhost:5004' : 'https://roar-server.herokuapp.com'
```

Comment:
This code technically works, but I feel like it's not consistent.
1. We use the `ENV` variable here to tell the user if we're running on 'local' or... whatever he wants? If I set the variable to 'hello_world' it will print 'hello_world'.
2. We use the `ENV` variable later in the rollup.config file and we check if the value is 'production', so the value isn't only for display. It actually change the build. We should be more specific here.
3. We can have the extension built for production but use the development server or built for development and use the production server. I can see the use case for each and I know we explicitly print that, but maybe we should issue a big red warning for production/development combinations. Also, if we don't specify both the ENV variable and the `--local-server` flag it'll build development but point to production. That seems counter-intuitive to me. Maybe we should have both `--local-server`\`--prod-server` and `--local-build`\`--prod-build`.